app/renderer/components/preferences/payment/ledgerBackup.js

```diff
@@ -74,7 +74,7 @@ class LedgerBackupFooter extends ImmutableComponent {
   }
 
   render () {
-    return <div className={css(paymentCommon.advanceFooter)}>
+    return <div className={css(commonStyles.modalOverlay__footer, commonStyles.modalOverlay__footerButton)}>
```

Comment:
Actually this kind of styling will be removed with the PR on refactoring modalOverlay, which I will submit later.